src/lt/plugins/vim.cljs

```diff
@@ -139,6 +139,13 @@
                     (object/raise cur :save)
                     (object/raise cur :close)))})
 
+(command {:command :vim-save-quit-x
+          :desc "Vim: :x"
+          :exec (fn []
```

Comment:
Since `:x` and `:wq` both do the same thing, could you replace the :exec function with a call to the `:wq` command? Something like:

```
:exec (fn []
            (cmd/exec! :vim-save-quit))
```

It should end up behaving exactly the same but cut down on duplicate code.